projects/vdk-control-cli/src/vdk/internal/control/command_groups/job/properties.py

```diff
@@ -327,13 +314,9 @@ def properties_command(
             )
 
 
-def _get_key_value_pairs(set, set_secret):
+def _get_key_value_pairs(set):
```

Comment:
Maybe this could be renamed to something like `set_property`?

Reply:
let's leave it as it is for now... pending refactoring.